### DISCUSSION TITLE
Linux and Docker Install

### DISCUSSION THREAD

**Discussion by mtoko** [frangoteam/FUXA#567] [General]
Hi

Thanks for all the great work on Fuxa. I have two questions, would appreciate some advice

Docker
I have successfully install a container in Docker and this all works except for the alarms. I have included a video demo of the issue

 

https://user-images.githubusercontent.com/4711179/198307133-3c8a017f-5892-406d-890b-dad1f6e100f3.mp4

Linux
I have tried to install on Ubuntu linux but get the following error
npm ERR! code 1
npm ERR! path /usr/lib/node_modules/@frangoteam/fuxa/node_modules/node-snap7
npm ERR! command failed
npm ERR! command sh -c -- prebuild-install || node-gyp rebuild
npm ERR! prebuild-install WARN install No prebuilt binaries found (target=16.18.0 runtime=node arch=x64 libc= platform=linux)
npm ERR! gyp info it worked if it ends with ok
Node version 16.18.0
NPM version 8.19.2

Any advice appreciated

**Comment by malcore10** [comment]
Hi, try to use FUXA Server Tags, internal device are only for frontend logic, alarms are handled by the server (backend)

[You need to have installed Node (Version 10, 12 or 14) and NPM (Version >=6.11 and < 7)](https://github.com/frangoteam/FUXA/wiki/Installing-and-Running)

**Reply by mtoko** [reply]
Thank you! 

Alarms are working correctly!

I will test the Linux install with the suggested Node and NPM versions
